coding-concepts.js

```diff
@@ -29,23 +29,23 @@ var languages = ["JavaScript", "Ruby", "Python", "C++"]
 var index = 1
 // console.log(languages[index])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: Ruby
+// b) Verify and explain: The answer was Ruby because the variable "index" is asking for the first word which is technically the second word because "Javascript" is the second word.
 
 
 // --------------------4) What will this log?
 
 var weekendDays = ["saturday", "sunday"]
 // console.log(weekendDays.toUpperCase())
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: Saturday, Sunday
+// b) Verify and explain: There was an error because you can not use the uppercase function on a variable. 
```

Comment:
We can use it on a variable but not on certain kinds. Whats happening here?

Reply:
This is an array so toUpperCase did not work.